Feature/updates (#74)
infinum/eightshift-docs: website/docs/basics/block-manifest.md

*custom feature*

This key gives you the ability to use component attributes in your block without mapping all the component's attributes every time. Please check [this chapter](blocks-component-in-block) for more details.

## The power of manifest.json

> As described before we use `manifest.json` to share stuff across multiple languages so you can easily see its power.

Reviewer: ```suggestion
As described before we use `manifest.json` to share stuff across PHP / JS / SCSS so you can easily see its power.
```

Reviewer 2: Also: 

```As described before, we use `manifest.json` to share stuff across PHP / JS / SCSS so you can easily see its power.```

The comma after before word.

> For example, you can use `manifest.json` to store SVG files that you will use on the frontend(PHP) and backend(JS) files. Or you can store block options so it is easier to find and add items.

Reviewer: ```suggestion
For example, you can use `manifest.json` to store SVG files that you will use on the frontend (PHP) and in the editor / backend (JS) files. Or you can store block options so it is easier to find and add items.
```

> Let's see how would you share SVG icon across multiple languages:

Reviewer 2: ```suggestion
Let's see how you would share SVG icon across multiple languages:
```
